core/web/schema/type/node.graphql

```diff
@@ -7,3 +7,26 @@ type Node {
     createdAt: Time!
     updatedAt: Time!
 }
+
+union NodePayload = Node | NotFoundError
+
+input CreateNodeInput {
+    name: String!
+    evmChainID: Int!
+    wsURL: String!
+    httpURL: String!
+    sendOnly: Boolean!
+}
+
+type CreateNodeSuccess {
+    node: Node!
+}
+
+union CreateNodePayload = CreateNodeSuccess
+
+
+type DeleteNodeSuccess {
+    node: Node!
+}
+
+union DeleteNodePayload = DeleteNodeSuccess | NotFoundError
```

Comment:
Mega small nit: no newline at the end of file. This is personal preference for me but I have VS Code ensuring a single newline at the end of each file. I believe with Intellij you can do that as well (Settings → Editor → General → Ensure line feed at file end on Save) but this is totally personal preference and we don't seem to enforce it in the codebase.